doc/user-guide/How-to-build.md

```diff
@@ -34,6 +34,13 @@ $ xcode-select --install # install compilation tools
 $ brew install git erlang openssl
 ```
 
+If you are planning to use ODBC to connect to MSSQL:
+
+```bash
+$ brew install unixodbc
```

Comment:
I think it's worth stating that `unixodbc` is required to compile MongooseIM and `freetds` is required to connect to MSSQL. Unless I'm mistaken.